tests/test_post.py

```diff
@@ -34,8 +34,12 @@
     from pyvista.plotting.renderer import CameraPosition
     from ansys.mapdl.core.theme import PyMAPDL_cmap
 
-from ansys.mapdl.core import examples
+from ansys.mapdl.core import _HAS_PYVISTA, examples
 from ansys.mapdl.core.errors import MapdlRuntimeError
+
+if _HAS_PYVISTA:
+    from ansys.mapdl.core.plotting.plotting import MapdlPlotter
```

Comment:
If using ``MapdlPlotter``, please remove the line 34 where ``from pyvista import Plotter``.
Furthermore, this line should be implemented in the above ``if has_dependency("pyvista"):`` context.

And then remove ``_HAS_PYVISTA``.

Reply:
Solved